platform/mikro-e/Makefile.mikro-e

```diff
@@ -42,7 +46,7 @@ endif
 
 MODULES += core/net core/net/mac core/net/llsec
 
-CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES)
+CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES) slip.c
```

Comment:
slip.c is Contiki common source code so it should be part of CONTIKI_SOURCEFILES which is independent of the platform. The platform specific source code will be part of CONTIKI_TARGET_SOURCEFILES.